descriptor.md

```diff
@@ -18,7 +18,8 @@ The following fields contain the primary properties that constitute a Descriptor
 
 - **`mediaType`** *string*
 
-  This REQUIRED property contains the media type of the referenced content.
+  This REQUIRED property MUST contain the media type of the referenced content unless this descriptor is the `config` property of an [image manifest](manifest.md).
```

Comment:
I don't think we need this with the addition of `artifactType` and the scratch media type.

Reply:
Removed